### PR TITLE
switches the integrated maneuvering module to use it's own gas tank

### PR DESCRIPTION
**Pull request by Mazianni**
The maneuvering thrusters module for RIGs now uses it's own gas tank.

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig/modules/utility.dm [hunk 1]
```diff
@@ -436,6 +452,11 @@
 	jets.holder = null
 	jets.ion_trail.set_up(jets)
 
+/obj/item/rig_module/maneuvering_jets/Destroy()
+	. = ..()
+	qdel(jets)
+	jets = null
```
Comment (quardbreak) [inline code/modules/clothing/spacesuits/rig/modules/utility.dm]: `QDEL_NULL(jets)`

**Review by quardbreak — CHANGES_REQUESTED** [review]

File: code/modules/clothing/spacesuits/rig/modules/utility.dm [hunk 2]
```diff
@@ -391,6 +391,22 @@
 	)
 	var/obj/item/tank/jetpack/rig/jets
 
+/obj/item/rig_module/maneuvering_jets/attackby(obj/item/W, mob/user)
+	. = ..()
```
Comment (MistakeNot4892) [inline code/modules/clothing/spacesuits/rig/modules/utility.dm]:
````
/obj/item/rig_module/maneuvering_jets/attackby(obj/item/W, mob/user)
  ...
  if(jets stuff)
    return TRUE
  . = ..()
  ````

**Review by MistakeNot4892 — DISMISSED** [review]

**Comment by Mazianni** [comment]
maybe going to cook up a custom sprite for the RIG jetpack so it's visually distinct

File: code/game/objects/items/weapons/tanks/jetpack.dm [hunk 3]
```diff
@@ -102,5 +102,7 @@
 	starting_pressure = list(/decl/material/gas/carbon_dioxide = 6*ONE_ATMOSPHERE)
 
 /obj/item/tank/jetpack/rig
-	name = "jetpack"
+	name = "integrated manuvering module thrusterpack"
+	desc = "The 'manuvering' part of an Manuvering jets module for a hardsuit. You could ... probably use this standalone?"
```
Comment (MistakeNot4892) [inline code/game/objects/items/weapons/tanks/jetpack.dm]: Random uppercase M? Also `...` shouldn't have a space before it.

Comment (MistakeNot4892) [reply code/game/objects/items/weapons/tanks/jetpack.dm]: Missed this yesterday sorry, `an maneuvering`.

File: code/game/objects/items/weapons/tanks/jetpack.dm [hunk 4]
```diff
@@ -102,29 +102,7 @@
 	starting_pressure = list(/decl/material/gas/carbon_dioxide = 6*ONE_ATMOSPHERE)
 
 /obj/item/tank/jetpack/rig
-	name = "jetpack"
+	name = "integrated manuvering module thrusterpack"
+	desc = "The 'manuvering' part of an manuvering jets module for a hardsuit. You could... probably use this standalone?"
```
Comment (MistakeNot4892) [inline code/game/objects/items/weapons/tanks/jetpack.dm]: `an maneuvering`

**Review by MistakeNot4892 — APPROVED** [review]
Needs squashmerge.

**Pull request merged**